examples/arm/setup.sh

```diff
@@ -167,7 +167,7 @@ function setup_tosa_reference_model() {
     # errors for the early codebase
     cd "${root_dir}"
     if [[ ! -e reference_model ]]; then
-        git clone https://review.mlplatform.org/tosa/reference_model -b v0.80.0
+        git clone https://review.mlplatform.org/tosa/reference_model -b 0.90.0d
```

Comment:
I see only [v0.90a0](https://review.mlplatform.org/plugins/gitiles/tosa/reference_model/+/refs/tags/v0.90a0) on https://review.mlplatform.org/plugins/gitiles/tosa/reference_model/, so the build is failing with `Remote branch 0.90.0d not found in upstream origin`.  Let me try to update the tag here to the correct value and rerun CI.

```suggestion
        git clone https://review.mlplatform.org/tosa/reference_model -b v0.90a0
```

Reply:
I tried it out with this `v0.90a0` branch and ended up with a slightly different version mismatch error git clone https://review.mlplatform.org/tosa/reference_model -b 0.90.0d

```
Read flatbuffer version 0.90.0d is not compatible with serializer version 0.80.0d (instead of 0.80.0)
```

cc @guangy10 Do you know where to find that version as it's not listed on https://review.mlplatform.org/plugins/gitiles/tosa/reference_model?

Reply:
Yea, `reference_model:v0.90a0` is still using the `0.80.0d` thirdparty serialization_lib which is causing the issue above. 

I changed to `git clone https://review.mlplatform.org/tosa/reference_model -b main` in the latest commit which is now currently using the 0.90.0d serialization_lib. https://review.mlplatform.org/plugins/gitiles/tosa/reference_model/+/refs/heads/main/thirdparty/serialization_lib

But it's still getting `ERROR: Read flatbuffer version 0.90.0d is not compatible with serializer version 0.80.0d`. Feels like the CI is still picking the v0.80 branch? 

Thanks for the help. This is a little bit convoluted now...

Reply:
I suspect this is due to the TODO that I mention in `.ci/docker/build.sh`, so I'm pushing a comment change there to trigger the rebuild.  Once the build is done https://github.com/pytorch/executorch/actions/runs/7453481163, could you help do another rebase or any minor changes to update the PR one more time.  That will pull the latest images from https://github.com/pytorch/executorch/actions/runs/7453481163.

Reply:
yes, for sure. I will try that once the new build is done. Thanks!

Reply:
It finally looks correct now https://github.com/pytorch/executorch/actions/runs/7454335305/job/20281469352?pr=1335, but I need to hardcode the Docker image to the correct version from https://github.com/pytorch/executorch/actions/runs/7453481163.  To unblock this change, let me import this and follow up on that Docker issue in a separate PR.  Does this sound good to you?

Reply:
Oh, I know a cleaner word around for now.  I could delete the upstream Docker image to ensure that the correct one is found and used instead of hard coding the version.  No action needed on your end though.  Please squash the commits as usual and I will reimport it.  Thank you for your patience :)

Reply:
no problem. thanks